src/main/java/exam2/lotto/LottoNumbers.java

```diff
@@ -0,0 +1,106 @@
+package exam2.lotto;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class LottoNumbers {
+
+    public final static int LOTTO_START_NUMBER = 1;
+    public final static int LOTTO_END_NUMBER = 45;
+    private List<List<Integer>> numberList = new ArrayList<>();
+
+    public LottoNumbers() {
+    }
+
+    //개수 만큼 랜덤 로또 번호를 생성합니다.
```

Comment:
설명이 필요한 주석은 javadoc 주석을 사용하는 편이에요~
```suggestion
/* 
* 개수 만큼 랜덤 로또 번호를 생성합니다.
*/
```